### ISSUE TITLE
Only cards with string matching the search query should be displayed.

### ISSUE THREAD

**Issue by aashish-ak** [aashish-ak/search_guru#31]
On searching for "Bha", this was the result:
![bhak](https://user-images.githubusercontent.com/21988675/50566057-59b91e80-0d5b-11e9-911b-86d445519852.png)
As "Bha" is only present in "Bhakti Joshi" 's name, so only her card should be visible.

On searching of "Bhak" , following was the result:
![bhak2](https://user-images.githubusercontent.com/21988675/50566101-9b49c980-0d5b-11e9-9763-21ee9b14a9be.png)
which is partially correct as "Ankush Bag" 's name does not contain "Bhak", so his card should not be there.

**Comment by dheeraj135** [comment]
Actually, this is because of fuzziness parameter in search query. Currently searching of name is done using fuzzy search. If we want partial matching, we can use [this](https://www.elastic.co/guide/en/elasticsearch/guide/current/partial-matching.html).

**Comment by aashish-ak** [comment]
Yes this is good, or this can also be done like using union of results of the string.includes() method and Elastic Search, are you willing to work on this?

**Comment by dheeraj135** [comment]
I will work on it, when I get some time.
